docs/management/managing-saved-objects.asciidoc

```diff
@@ -61,7 +61,7 @@ You have two options for exporting saved objects.
 * Click *Export x objects*, and export objects by type.
 
 This action creates an NDJSON with all your saved objects.  By default,
-the NDJSON includes related objects. Exported dashboards include their associated index patterns.
+the NDJSON includes related child objects. Exported dashboards include their associated index patterns.
```

Comment:
We don't really say "related child objects" anywhere else in Kibana as far as I can tell. The only place where child/parent relations appear to be described is in the Saved Objects Management page (click a row action to view related objects).

I figure that for the purposes of docs, "related child objects" is fine, unless someone has a better idea?

Reply:
How about:

By default, the NDJSON includes child objects that are related to the saved objects.